### PR TITLE
Upgrade Plan

### PR DESCRIPTION
**Pull request by achaparro**
Hi @jhinkey,

Sorry for the delay.

The changes in the documentation look fantastic!

I added some minor clarifications, please review them. And the only I see a bit confused is this article:
https://github.com/jhinkey/liferay-docs/pull/446/files#diff-d46da70a17875dfc94bca8c9b734a9fc

Maybe we can talk tomorrow about it although maybe it's just because the process itself is a bit complex.

Thanks so much for your work. This is going to be really helpful for customers and community :-)

Cheers!

### CODE REVIEW

File: deployment/articles/05-upgrading-to-liferay-7-2/06-upgrading-the-liferay-database/01-upgrading-the-liferay-database-intro.markdown [hunk 1]
```diff
@@ -25,15 +25,19 @@ If you are upgrading from Liferay Portal 6.2 or earlier, it's recommended to use
...
+processes in the Core (database schema micro version changes) are not
+mandatory. This means you can install a fix pack (i.e., core code) without
+having to execute the database schema micro version changes---you can execute
+micro version changes when you want, even outside of a major or minor version
+upgrade. Before using the upgrade tool to execute a fix pack's micro upgrade
+process, however, you must shut down the server, install the fix pack, and 
 [back up the @product@ database, installation, and Document Library store](/discover/deployment/-/knowledge_base/7-1/backing-up-a-liferay-installation). 
+
+For modules, micro database schema version changes are executed after fix pack
+installation unless you set AutoUpgrade config to false.
```
Comment (jhinkey) [inline deployment/articles/05-upgrading-to-liferay-7-2/06-upgrading-the-liferay-database/01-upgrading-the-liferay-database-intro.markdown]: @achaparro Will module micro database schema version changes in fix packs execute automatically after fix pack installation unless AutoUpgrade is set to false?

Thanks!

Comment (achaparro) [reply deployment/articles/05-upgrading-to-liferay-7-2/06-upgrading-the-liferay-database/01-upgrading-the-liferay-database-intro.markdown]: Yes, that's it!

Comment (achaparro) [reply deployment/articles/05-upgrading-to-liferay-7-2/06-upgrading-the-liferay-database/01-upgrading-the-liferay-database-intro.markdown]: The upgrade is executed on server startup

**Comment by jhinkey** [comment]
Started reviewing.

**Comment by jhinkey** [comment]
I wordsmithed your changes and posted them here: https://github.com/jhinkey/liferay-docs/tree/72-upgrading-liferay/deployment/articles/05-upgrading-to-liferay-7-2

I'm going to use fresh branches based on master, to send each article for final review and publishing. I'll also update some of the Markdown to use our latest conventions and features. 

Thanks!

**Comment by achaparro** [comment]
Thanks @jhinkey!

**Pull request closed without merging**